CHANGES.md

```diff
@@ -29,6 +29,11 @@ OpenSSL 3.0
 
    *Tomas Mraz*
 
+ * Deprecate EVP_MD_CTX_set_update_fn() and EVP_MD_CTX_update_fn()
+   although they are still used internally in legacy (engine) code.
```

Comment:
I am not sure the `although they are still used internally in legacy (engine) code.` is really bringing any important information to the user. I think something like `as they are not useful with non-deprecated functionality.` would be better. Or something in that sense.

Reply:
I'm sorry for being picky.

Reply:
I thought what I had gave more info, but I don't care very much.  Wording changed and updated commit pushed.